functional_tests/scylla_operator/test_functional.py

```diff
@@ -121,3 +121,16 @@ def test_mgmt_backup(tester, db_cluster):
     mgr_task = mgr_cluster.create_backup_task(location_list=[bucket_name, ])
     status = mgr_task.wait_and_get_final_status(timeout=54000, step=5, only_final=True)
     assert TaskStatus.DONE == status
+
+
+def test_listen_address(tester, db_cluster):
+    """
+    Issues: https://github.com/scylladb/scylla-operator/issues/484
+            https://github.com/scylladb/scylla/issues/8381
+    Fix commit: https://github.com/scylladb/scylla-operator/pull/529
+    """
+    for node in db_cluster.nodes:
+        with node.remote_scylla_yaml() as scylla_yaml:
+            listen_address = scylla_yaml.get('listen_address')
+            assert listen_address, "Not found listen_address flag"
+            assert listen_address == '0.0.0.0', f'Unexpected {listen_address} for {node.name} node'
```

Comment:
It is better to dump all error messages into the list and assert on it

```
    errors = []

    for node in db_cluster.nodes:
        with node.remote_scylla_yaml() as scylla_yaml:
            if not listen_address: 
                errors.append(f"Node {node.name } has no listen_address in scylla.yaml")
            elif listen_address != "0.0.0.0":
                errors.append(f"Node {node.name } has wrong listen_address "{listen_address}" in scylla.yaml")
    assert not errors, "Following errors found:\n{'\n'.join(errors)}"

```

Reply:
fixed